apps/public-docsite-v9/src/shims/MenuShim.tsx

```diff
@@ -54,7 +55,7 @@ const shimMenuHeaderProps = (props: IContextualMenuItem): MenuGroupHeaderProps =
 export const MenuItemShim = (props: IContextualMenuItem) => {
   if (props.itemType === ContextualMenuItemType.Divider) {
     const shimProps = shimMenuItemProps(props);
-    return <MenuDivider {...shimProps} />;
+    return <MenuDivider {...(shimProps as MenuDividerProps)} />;
```

Comment:
I don't understand how the changes you made caused this cast to be required, but it won't hurt :)

Reply:
The casting is required since spreading of props can be conflicting when using discriminated unions. Basically there's no way to know if we're spreading `button` props or `a` props, so we need to explicitly upper cast to ensure it's a generic that represents both.